docs/user_guide/install_tripal/manual_install/install_drupal.rst

```diff
@@ -78,11 +85,11 @@ If an index.html file is present (as is the case with Ubuntu installations) you
 
 .. note::
 
-  It is extremely important the the hidden file .htaccess is also moved (note the second 'mv' command above. Check to make sure this file is there
+  It is extremely important the the hidden file ``.htaccess`` is also moved (note the second ``mv`` command above. Check to make sure this file is there:
 
-.. code-block:: bash
+  .. code-block:: bash
```

Comment:
Note this isnt whitespace changing: this moves the codeblock *INSIDE* the note.  I think this is more appropriate.